### PR TITLE
Issues/#381

### PR DESCRIPTION
**Pull request by camskene**
This pull request remove the `bind-attr` syntax.

### CODE REVIEW

File: app/templates/deprecations.hbs [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
 
-              <span {{bind-attr title="message" class="item.expandedClass"}} {{action "toggleExpand"}}  >
+              <span title="{{message}}" class="{{item.expandedClass}}" {{action "toggleExpand"}}  >
                 <span class="cell__arrow"></span>
                 <span class="pill pill_not-clickable" data-label="deprecation-count">{{item.model.count}}</span>
                 <span data-label="deprecation-message" data-label="deprecation-message">{{item.model.message}}</span>
               </span>
               {{#if item.model.url}}
-                <a {{bind-attr href=item.model.url}} class="external-link" target="_blank" title="Transition Plan" data-label="deprecation-url">
+                <a href={{item.model.url}} class="external-link" target="_blank" title="Transition Plan" data-label="deprecation-url">
```
Comment (teddyzeenny) [inline app/templates/deprecations.hbs]: Can you add quotes here? `href="{{item.model.url}}"`

**Comment by teddyzeenny** [comment]
Thank you!! Do you know why the tests are failing?

**Comment by camskene** [comment]
I don't understand why those tests are failing.

**Comment by teddyzeenny** [comment]
We need to upgrade Ember to 1.11.2. I will try to work on that when I find some time.

**Comment by teddyzeenny** [comment]
Upgraded to Ember 1.12. Can you try to rebase?

**Comment by rwjblue** [comment]
Sorry for the run around here. Can you rebase and squash commits?

**Comment by knownasilya** [comment]
Looks done.

**Comment by camskene** [comment]
Yes, all done. Would be great to see this merged in :) Let me know if you need anything else here. Thanks.

**Comment by locks** [comment]
@camskene thank you for your work, seems like it was fixed in the meantime!

**Pull request closed without merging**